sdk/highlight-next/src/util/withHighlightConfig.ts

```diff
@@ -69,6 +69,7 @@ const getDefaultOpts = (
 	}
 }
 
+// TODO(vkorolik) allow config to be a fn
```

Comment:
why is that?

Reply:
ran into an issue with runway's setup where `withHighlight(withSentry(...))` was broken, because `withSentry` returned a function. the type def for `NextConfig` changes in v13 to allow this. thanks for catching the TODO - will finish that 

![Screenshot 2023-02-02 at 12 10 30 PM](https://user-images.githubusercontent.com/1351531/216438923-f4c97004-58fb-4274-bdfd-3e0a347c3cad.png)